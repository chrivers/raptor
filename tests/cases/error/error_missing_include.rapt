# line 1
INCLUDE missing_file
# line 3
