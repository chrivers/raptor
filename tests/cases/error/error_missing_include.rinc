error_missing_include.rapt