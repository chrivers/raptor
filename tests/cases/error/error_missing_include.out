error: Error while loading source file
 --> tests/cases/error/error_missing_include.rapt:1:1
  |
1 | INCLUDE missing_file
  | ^^^^^^^^^^^^^^^^^^^^ Could not open [tests/cases/error/missing_file.rinc]: No such file or directory (os error 2)
2 |
  |
